packages/v4/patternfly-docs/pages/home.js

```diff
@@ -10,269 +25,302 @@ import {
-            </Text>
-            <div className="pf-v5-u-display-flex pf-v5-u-justify-content-center pf-v5-u-flex-direction-column pf-v5-u-flex-direction-row-on-md">
+            <Flex
+              direction={{default: "column"}}
+              spaceItems={{default: "spaceItems2xl"}}
+            >
+              <Title headingLevel="h1" size="4xl">
+                <Text className="pf-v5-u-content-bold">
+                  {" "}
+                  Design and build better product experiences in the
+                  <Text component="span" className="pf-v5-u-primary-color-100">
+                    {" "}
+                    open{" "}
+                  </Text>
+                  with PatternFly
+                </Text>
+              </Title>
+              <Text component={TextVariants.p} className="ws-mdx-p">
+                PatternFly is an open source design system that enables teams to
+                create consistent and scalable enterprise products. PatternFly
+                is sponsored and maintained by Red Hat, but is open to all.
+              </Text>
+            </Flex>
+          </TextContent>
+          <Flex
+            flexWrap={{default: "nowrap"}}
+            spaceItems={{default: "spaceItemsMd"}}
+          >
+            <FlexItem>
               <Link
-                to="/get-started/about"
-                className="pf-v5-c-button pf4-c-button__cta-primary pf-v5-u-mr-0 pf-v5-u-mb-md pf-v5-u-mb-0-on-md pf-v5-u-mr-xl-on-md fadeIn animated fadeInFour"
-                aria-label="Get started with PatternFly"
+                to="/get-started/design"
+                className="pf-v5-c-button pf-m-secondary pf-m-display-lg"
+                aria-label="Start designing"
               >
-                Get started
+                Start designing
               </Link>
-              <a className="pf-v5-c-button pf4-c-button__cta-outline fadeIn animated fadeInFour" href="//github.com/patternfly" target="_blank">
-                <GithubIcon className="pf-v5-u-mr-sm" />GitHub
-              </a>
-            </div>
-          </TextContent>
-        </GridItem>
-      </Grid>
+            </FlexItem>
+            <FlexItem>
+              <Link
+                to="/get-started/develop"
+                className="pf-v5-c-button pf-m-secondary pf-m-display-lg"
+                aria-label="Start developing"
+              >
+                Start developing
+              </Link>
+            </FlexItem>
+          </Flex>
+        </Flex>
+        <Flex flex={{default: "flex_4"}}>
+          <img className="pf-v5-c-image__light-showcase" src={lightshowcase} />
+        </Flex>
+      </Flex>
     </PageSection>
-    <PageSection className="ws-homepage-main-section pf-m-light-100">
-      <Grid>
-        <GridItem sm={12} md={6} mdOffset={3} className="pf-v5-u-py-2xl pf-v5-u-text-align-center" id="about-patternfly-section">
+    <Divider />
+    <PageSection>
+      <Flex flexWrap={{default: "wrap"}}>
```

Comment:
I didn't see the design, but seems like the title and link are awfully close to the cards. Maybe a stack with a gutter would be good to have to provide that?
I see that the cards get very narrow before wrapping. Also, a gallery will make it easier for the cards to stretch vertically. I think this is a good place to use a gallery layout.
What do you think about laying this section out something like this?
```
stack pf-m-gutter
    split pf-m-gutter
        title pf-m-fill
        link button
    gallery pf-m-gutter
        card
        card
```